packages/patterns-react/package.json

```diff
@@ -49,7 +49,11 @@
     "@babel/runtime": "^7.5.5",
     "@carbon/ibmdotcom-services": "0.4.0",
     "@carbon/ibmdotcom-styles": "0.4.0",
+<<<<<<< Updated upstream
```

Comment:
@sannminwin this looks like this was accidentally merged in

Reply:
ah, I will fix it now.

Reply:
corrected